docs/release-notes/rockcraft-1-16.rst

```diff
@@ -0,0 +1,150 @@
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.16 brings the following features, integrations, and improvements.
+
+
+``ubuntu@26.04`` base
+~~~~~~~~~~~~~~~~~~~~~
+
+Rockcraft now supports ``ubuntu@26.04`` as a base. Since this version of Ubuntu is in
+development at time of release, to select this base you must also set
+``build-base: devel``.
+
+
+Stabilized ``ubuntu@25.10`` base
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``ubuntu@25.10`` base is now fully supported and no longer requires ``build-base:
+devel``.
+
+
+Availability on RISC-V 64-bit and armhf
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Rockcraft is once again available on RISC-V 64-bit and armhf systems.
+
+
+JLink plugin
+~~~~~~~~~~~~
+
+The :ref:`craft_parts_jlink_plugin` supports two new keys.
+
+- ``jlink-modules`` declares the complete list of modules to include in the OpenJDK
+  image
+- ``jlink-multi-release`` specifies the OpenJDK release version to use for multi-release
+  JARs
+
+
+Minor features
+--------------
+
+Rockcraft 1.16 brings the following minor changes.
+
+
+Improved ``rockcraft.yaml`` Documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We refreshed the rockraft.yaml reference. The page
+covers all keys, and their descriptions and examples are now sourced from the codebase,
+so they will stay up-to-date as Rockcraft changes.
+
+The reference has also been reorganized to better reflect idiomatic project file
+structure and the new ``platforms`` keys.
+
+
+Documentation for 12-factor app extensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`set-up-web-app-rock` now contains the steps needed to override the rock's default
+command that the 12-factor app extensions provide.
+
+
+Backwards-incompatible changes
+------------------------------
+
+The following changes are incompatible with previous versions of Starcraft.
+
+
+Removed character support in platform names
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Platform names no longer support forward slashes (/).
+
+
+Fixed bugs and issues
+---------------------
+
+The following issues have been resolved in Rockcraft 1.16.
+
+
+Rockcraft 1.16.0
+~~~~~~~~~~~~~~~~
+
+- `#992 <https://github.com/canonical/rockcraft/issues/992>`_ Architecture field in
+  ``.rock/metadata.yaml`` is serialized incorrectly.
+- `#1022 <https://github.com/canonical/rockcraft/issues/1022>`_ ``entrypoint-service``
```

Comment:
This one isn't actually closed. Is it complete?

Reply:
yeap, it's on my list to merge before merging this